Detectors/MUON/MCH/PreClustering/src/PreClusterFinder.h

```diff
@@ -68,16 +68,13 @@ class PreClusterFinder
  private:
   struct DetectionElement {
     std::unique_ptr<Mapping::MpDE> mapping; // mapping of this DE including the list of pads
-    std::vector<const DigitStruct*> digits; // list of pointers to digits (not owner)
+    std::vector<const Digit*> digits;       // list of pointers to digits (not owner)
```

Comment:
again, if the vector is not owner, then it's probably a span, isn't it ? (and thus the comment "not owner" would not be needed at all)

Reply:
Not here. The vector is not owner of the digits but it is owner of the pointers. It cannot be a span because the digits are sorted per detection element when they are loaded in the preclustering, while they are not in the input message. Well they probably are in my tests but this is not guarantee a priori for the moment.